README.md

```diff
@@ -17,41 +19,31 @@
 * Create as many as desired with easy polyzone creation
 * Vehicle sale amount gets deposited into the cardealer society fund for private dealer
 
-**Planned Updates**
-* qbx_phone support to make payments
-
-**Preview header when near a vehicle at the public dealership:**
-
-![image](https://user-images.githubusercontent.com/57848836/138773379-836be2a6-a800-47a4-8037-84d9052a964c.png)
-
-**After pressing the focus key and selecting the preview header (default: LEFT ALT)**
+**Vehicle Configuration System**
+* Model Check: The system first looks for the vehicle model in the models table. If a matching model is found, its will be displayed.
 
-![image](https://user-images.githubusercontent.com/57848836/138770886-15e056db-3e57-43ea-b855-3ef4fd107acf.png)
+* Category Check: If the vehicle model is not found, the system then checks the categories table to see if a category matches the vehicle. Its will be displayed in menu if a match is found.
```

Comment:
This seems like a verbose and confusing explanation. I would just summarize the feature as can specify default vehicle shops for vehicles, or default all vehicles of a certain category to specific vehicle shops, with overrides/exceptions to reduce configuration file sizes.